natsjobs/config.go

```diff
@@ -42,8 +42,8 @@ func (c *config) InitDefaults() {
 		c.Priority = 10
 	}
 
-	if c.Stream == "" {
-		c.Stream = "default-stream"
+	if c.StreamID == "" {
```

Comment:
We can leave `c.StreamID` here btw. But could you please use `default-stream` as the default name for the stream?